include/s3select.h

```diff
@@ -2174,6 +2176,8 @@ class base_s3object
   scratch_area* m_sa;
   std::string m_obj_name;
   bool m_aggr_flow = false; //TODO once per query
+  bool is_star = false;
```

Comment:
seems a redundant data-member.
there is `bool star_operation_ind;` on `json_object`